Desctiption: use packaged version of gmm instead of shipped with tarball
 Patch needs to be applied, when getfem++ will be able to migrate to testing.
 Requires BD on libgmm++-dev
Author: Christophe Trophime <christophe.trophime@lncmi.cnrs.fr>
Reviewed-by: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-01-06

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -481,8 +481,10 @@
 endif(ENABLE_DINTEGRATION)
 
 if(ENABLE_GMM)
-  include_directories(contrib/gmm)
-  set_config_option(HAVE_GMM "Gmm")
+  find_path(GMM_INC "gmm.h" PATH_SUFFIXES src include include/gmm)
+  if(GMM_INC)
+    list(APPEND EXTERNAL_INCLUDES ${GMM_INC})
+  endif(GMM_INC)
 endif(ENABLE_GMM)
 
 if(ENABLE_KBIPACK)
